ports/stm32/boards/WEACT_F411_BLACKPILL/mpconfigboard.mk

```diff
@@ -0,0 +1,26 @@
+MCU_SERIES = f4
+CMSIS_MCU = STM32F411xE
+AF_FILE = boards/stm32f411_af.csv
+LD_FILES = boards/stm32f411.ld boards/common_ifs.ld
+TEXT0_ADDR = 0x08000000
+TEXT1_ADDR = 0x08020000
+
+MICROPY_VFS_LFS2 = 1
+
+# Settings for spi flash and HSE xtal.
+
+ifdef SPI_FLASH_SIZE_MB
+# Add flash size to build folder name if set from command line
+BUILD := $(BUILD)_FLASH_$(SPI_FLASH_SIZE_MB)M
+endif
+
+ifdef XTAL_FREQ_MHZ
+# Add xtal speed to build folder name if set from command line
+BUILD := $(BUILD)_XTAL_$(XTAL_FREQ_MHZ)M
+endif
```

Comment:
Because of the makefile ordering this isn't used / run when a variant is configured - this is just for when one of these settings is passed on the `make` command line. I thought this would be most helpful for people manually configuring the FLASH size from command line, it will put the flash size in the build folder name.

Reply:
Could I suggest a warning in the README.md to not use A4, A5, A6 or A7 on the 8MB variant.  I was vaguely aware that those pins were used for the SPI flash, but did not appreciate the consequences of "accidentally" using them.

Reply:
I've added a note to the [readme](https://github.com/micropython/micropython/blob/2b7cd314ad22ba65bb249d82d72deef32b09c679/ports/stm32/boards/WEACT_F411_BLACKPILL/README.md) that's a good suggestion; it's not immediately obvious that the pins needed for "on-board" functions are also exposed to the end user.